# Type: test
# Subtype: T1
# English Summary: Re-enable a test
# Chinese Summary: 重新启用一个测试
# Entry Number: 7

# === Git Diff Content ===

diff --git a/sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java b/sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java
index 0d611c38b..424cb190c 100644
--- a/sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java
+++ b/sql/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java
@@ -118,7 +118,6 @@ public class TransportSQLActionTest extends SQLTransportIntegrationTest {
         assertEquals(1L, response.rows()[0][0]);
     }
 
-    /*
     @Test
     public void testGroupByOnSysNodes() throws Exception {
         execute("select count(*), name from sys.nodes group by name");
@@ -127,7 +126,6 @@ public class TransportSQLActionTest extends SQLTransportIntegrationTest {
         execute("select count(*), hostname from sys.nodes group by hostname");
         assertThat(response.rowCount(), is(1L));
     }
-    */
 
     @Test
     public void testSysCluster() throws Exception {
